specs/linux/ipmi_lan.table

```diff
@@ -0,0 +1,19 @@
+table_name("ipmi_lan")
+description("IPMI LAN information.")
+schema([
+    Column("mc_name", TEXT, "Name of the MC"),
```

Comment:
I wanted to be specific in non `ipmi_mc` tables, that the `mc_name` can be joing with `name` of `ipmi_mc`...